scripts/globals/items/pinch_of_rainbow_powder.lua

```diff
@@ -2,25 +2,23 @@
 -- ID: 5362
 -- Rainbow Powder
 -- When applied, it makes things invisible.
+-- Removed Medicated status as per https://www.bg-wiki.com/ffxi/Rainbow_Powder
 -----------------------------------
 require("scripts/settings/main")
 require("scripts/globals/status")
-require("scripts/globals/msg")
 -----------------------------------
 local item_object = {}
 
 item_object.onItemCheck = function(target)
-    if (target:hasStatusEffect(xi.effect.MEDICINE)) then
-        return xi.msg.basic.ITEM_NO_USE_MEDICATED
-    end
     return 0
 end
 
 item_object.onItemUse = function(target)
-    local duration = 600
-    target:delStatusEffect(xi.effect.INVISIBLE)
-    target:addStatusEffect(xi.effect.INVISIBLE, 0, 10, math.floor(duration * xi.settings.SNEAK_INVIS_DURATION_MULTIPLIER))
-    target:addStatusEffect(xi.effect.MEDICINE, 0, 0, 180)
+    if (target:hasStatusEffect(xi.effect.INVISIBLE)) then
```

Comment:
this check isn't needed, there is no harm, nor is the delete, invisible already overwrites itself